harvester_e2e_tests/conftest.py

```diff
@@ -210,9 +210,21 @@ def pytest_addoption(parser):
     parser.addoption(
         '--kubernetes-version',
         action='store',
-        default=config_data.get('kubernetes-version', 'v1.21.6+rke2r1'),
+        default=config_data.get('kubernetes-version', 'v1.24.11+rke2r1'),
```

Comment:
Can we just use the value from config.yml and not set a fallback value here? Reason being as below:
1. Kubernetes version keeps updating frequently, we'll have overhead of updating the versions at 2 places.
2. It is okay to let the test fail if versions are not defined properly in config.yml or we have some problem in parsing config.yml.
3. Let these values in the file be consistent with others as we are not putting fallback values for other variables.

Reply:
updated.